packages/block-editor/src/components/url-input/index.js

```diff
@@ -84,12 +84,13 @@ class URLInput extends Component {
 			}, 100 );
 		}
 
-		// Only attempt an update on suggestions if the input value has actually changed.
 		if (
 			prevProps.value !== value &&
-			this.shouldShowInitialSuggestions()
+			! this.props.disableSuggestions &&
+			! this.isUpdatingSuggestions &&
+			! this.props.__experimentalShowInitialSuggestions
 		) {
-			this.updateSuggestions();
+			this.updateSuggestions( value.trim() );
```

Comment:
Could you remind me as to the value of `trim()` here?

Reply:
All the other calls for `this.updateSuggestions` are made with `value.trim()`.
Ideally, the trimming should be done inside `updateSuggestions`, so that the function could just be called with value: `this.updateSuggestions(value)`.

Reply:
Ok let's leave "as is" in this PR and address moving the `.trim()` operation to the `updateSuggestions` method in a follow up PR. Would you be able to handle that or would you like me to take care of it?